examples/using-sqip/src/pages/index.js

```diff
@@ -173,9 +173,7 @@ export const query = graphql`
           publicURL
```

Comment:
Note to myself: the text on the actual using-sqip website still references to gatsby-image